bower.json

```diff
@@ -29,6 +29,7 @@
     "core-doc-viewer": "Polymer/core-doc-viewer#0.8-preview",
     "core-icon": "Polymer/core-icon#0.8-preview",
     "core-icons": "Polymer/core-icons#0.8-preview",
+    "test-fixture": "Polymer/test-fixture#master",
```

Comment:
In principle I think we should specify a version, but in practice is the plan to update all dependent elements when we rev the version of some element?